Assets/HotmaxEditorVR/objectSelect.cs

```diff
@@ -5,174 +5,158 @@
 
 public class objectSelect : MonoBehaviour
 {
-    public Valve.VR.InteractionSystem.Hand hand1;
-    public Valve.VR.InteractionSystem.Hand hand2;
+    //--laser stuff
+    private Vector3 laserEndPosition;
+    public LineRenderer laserLineRenderer;
```

Comment:
make 10 private?

Reply:
it has to be public so other classes can grab it. we are using the hand2 in "universalTransform" to fetch the gameobjects worldspace transform.  We won't be doing this in the future. maybe we can change it to private later. ill put a todo there